### PR TITLE
Daily and dailyedit cleanup

### PR DESCRIPTION
**Pull request by Drummss**

### CODE REVIEW

File: src/Miki/Modules/Accounts/AccountsModule.cs [hunk 1]
```diff
@@ -1191,14 +1193,17 @@ public async Task DailyStatsAsync(IContext e)
...
 
-                embed.AddInlineField(e.GetLocale().GetString("daily_stats_current_streak"), $"{daily.CurrentStreak}");
-                embed.AddInlineField(e.GetLocale().GetString("daily_stats_longest_streak"), $"{(daily.LongestStreak >= daily.CurrentStreak ? daily.LongestStreak:daily.CurrentStreak)}");
-                embed.AddInlineField(e.GetLocale().GetString("daily_stats_time"), $"{time}");
+                var embed = new EmbedBuilder()
+                    .SetTitle(locale.GetString("daily_stats_title", $"{e.GetAuthor().Username}"))
+                    .SetColor(253, 216, 136)
+                    .AddInlineField(locale.GetString("daily_stats_current_streak"), $"{daily.CurrentStreak}")
+                    .AddInlineField(locale.GetString("daily_stats_longest_streak"), 
+                        $"{(daily.LongestStreak >= daily.CurrentStreak ? daily.LongestStreak:daily.CurrentStreak)}")
```
Comment (velddev) [inline src/Miki/Modules/Accounts/AccountsModule.cs:1205]: This already happens in the service layer. Seems odd to do it twice

**Review by velddev — APPROVED** [review]
Left one comment, but will approve.

File: src/Miki/Modules/Accounts/AccountsModule.cs (hunk 1 above)
Comment (Drummss) [reply src/Miki/Modules/Accounts/AccountsModule.cs:1205]: Yeah, I left it as an edge case incase the current streak gets changed without the longest streak being changed as well

Comment (velddev) [reply src/Miki/Modules/Accounts/AccountsModule.cs:1205]: But this never happens, right?

**Pull request merged**